sql/returning.go

```diff
@@ -34,8 +34,8 @@ type returningHelper struct {
 }
 
 func makeReturningHelper(p *planner, r parser.ReturningExprs,
-	alias string, tablecols []ColumnDescriptor) (*returningHelper, error) {
-	rh := &returningHelper{p: p}
+	alias string, tablecols []ColumnDescriptor) (returningHelper, error) {
+	rh := returningHelper{p: p}
```

Comment:
this is doing that thing again - a later commit reverting an earlier commit's change.